### ISSUE TITLE
Translation into other languages

### ISSUE THREAD

**Issue by qWici** [k88hudson/git-flight-rules#162]
How about creating a project on [Crowdin ](https://crowdin.com/) and maintain the translation into other languages. We can choose several people who are fluent in some languages and who will follow the actuality of the translation. Because as I see the process of translating into Spanish is very slow #119 

---
**UPD:**
If you want help with translation please follow [this guide](https://github.com/k88hudson/git-flight-rules/blob/master/CONTRIBUTE.md#translation)

**Comment by RichardLitt** [comment]
I don't have a budget for that - this is entirely volunteer work. I wouldn't want to put this on anyone else, either. Perhaps there is a free solution for GitHub that we could look into?

**Comment by buslov** [comment]
There is [GitLocalize](https://gitlocalize.com). It have good integration with GitHub and it's free for public repos. I've tested it for my repo [GitHub](https://github.com/buslov/test) / [GitLocalize](https://gitlocalize.com/repo/568)

**Comment by qWici** [comment]
@RichardLitt Hmm..interesting. Since this question is not urgent, volunteers can translate. Create a new project and leave a link here.

**Comment by RichardLitt** [comment]
Looks like only @k88hudson can do this. :)

**Comment by eamanu** [comment]
Sound very interest {(https://crowdin.com/)

**Comment by sotayamashita** [comment]
@qWici Hi, I am engineer for GitLocalize. If you have any question. Please feel free to ask me.

**Comment by RichardLitt** [comment]
@sotayamashita Is there any way for non-admin's to enable it?

**Comment by sotayamashita** [comment]
@RichardLitt You it is. You can translate docs if admin invite on GitLocalize.

**Comment by RichardLitt** [comment]
@sotayamashita I'm not sure I'm seeing how to add it. [On this page](https://gitlocalize.com/repo/new), I'm not seeing k88hudson/git-flight-rules. But I'm not an admin on this repo, just another maintainer, so it may not be possible for you to enable it with my permission set. If it is, though, that would be great.

**Comment by sotayamashita** [comment]
@RichardLitt You need to ask repository owner to integrate on GitLocalize.

**Comment by RichardLitt** [comment]
@k88hudson Second ping. :)

**Comment by k88hudson** [comment]
Oops sorry, I'm on it!

**Comment by k88hudson** [comment]
Ok, I've added the repository. I'm getting an error code when trying adding moderators/admins though, perhaps you need to give authorize GitLocalize first?

**Comment by sotayamashita** [comment]
@k88hudson 

> do users need to authorize GitLocalize first

Yes. they need to signup at GitLocalize first to add them as a moderators.

**Comment by jpmakangiras** [comment]
@k88hudson May i translate this into bahasa indonesia?

**Comment by Forbrig** [comment]
I would like to help with the translation to _portuguese-br_!

**Comment by RichardLitt** [comment]
@jpmakangiras and @Forbrig: that'd be great! @sotayamashita, can @k88hudson make me an admin so that I can add people?

**Comment by sotayamashita** [comment]
@RichardLitt 

> @sotayamashita, can @k88hudson make me an admin so that I can add people?

Yes. you can do it. :+1:

**Comment by RichardLitt** [comment]
@k88hudson Try adding me, then. :)  https://gitlocalize.com/repo/598/roles

**Comment by k88hudson** [comment]
@sotayamashita I am still getting a 404 error when trying to add `RichardLitt`as an admin, any ideas why that might be happening?

**Comment by sotayamashita** [comment]
@k88hudson I am sorry. I have added `RichardLittas` as an admin. I will look into it.

**Comment by RichardLitt** [comment]
@eamanu @jpmakangiras @Forbrig We'll need you to have README files for your languages, first. Let's go!

@qWici Sign up on GitLocalize, please. :) @buslov Added you

**Comment by sotayamashita** [comment]
@RichardLitt I saw your repository on GitLocalize. It is conflicted. Please see https://docs.gitlocalize.com/how_to_contribute.html#5-resolve-a-conflict If you have any question, feel free to ask me on here or intercom on GitLocalize

**Comment by RichardLitt** [comment]
Resolving the conflicts seems to require that I link up every section for the entire document, manually. That sounds very lengthy and arduous. Am I missing an easier way?

**Comment by sotayamashita** [comment]
@RichardLitt 

> That sounds very lengthy and arduous. Am I missing an easier way?

Sorry we do not have :cry:

**Comment by Forbrig** [comment]
@sotayamashita @RichardLitt 

> @eamanu @jpmakangiras @Forbrig We'll need you to have README files for your languages, first. Let's go!

I have forked this repo and addeded it to **Gitlocalize** but when I submit a string I get a warning: "_Content You need to upgrade the plan to translate_". What should I do? (Already have send an email asking for upgrade).

**Comment by eamanu** [comment]
Hi everybody, I was a little busy at work. I will work in the translation to spanish. @RichardLitt @k88hudson @moisesnandres.

Regards!

**Comment by qWici** [comment]
@RichardLitt I logged in. You move)
[My profile](https://gitlocalize.com/users/qwici)

**Comment by sotayamashita** [comment]
@Forbrig You do not need to integrate with forked repository on GitLocalize.

@RichardLitt Could you invite @Forbrig over Gitlocalize?

**Comment by jpmakangiras** [comment]
@RichardLitt Could you also invite me?

**Comment by qWici** [comment]
@RichardLitt @k88hudson Also, look what I found. Maybe it will be useful in the future. 
https://github.com/facebook/Docusaurus

**Comment by sotayamashita** [comment]
@jquintus I think Docusaurus is very good but it is too much for it.

**Comment by wviana** [comment]
I would like to help @Forbrig translating into PT-BR.

We should add an how to translate in the beginning of README.

**Comment by sotayamashita** [comment]
@wviana I have added the description on CONTRIBUTING.md. De we need to do it too?

**Comment by Forbrig** [comment]
@sotayamashita 

Maybe add some steps between item _2._ and _3._ about contacting the moderator (maybe @RichardLitt  if he don't mind) to be invited to the repo on **gitLocalize**. Showing the info that I misunderstood and avoiding that this issue become a chat...

**Comment by aloisdg** [comment]
I started a fr translation: https://github.com/k88hudson/git-flight-rules/pull/189

**Comment by eamanu** [comment]
@k88hudson  @RichardLitt I need you that open me a spanish README in GitLocalize for me

**Comment by qWici** [comment]
@aloisdg do not forget generate menu with DocToc

**Comment by RichardLitt** [comment]
@eamanu done
@sotayamashita - spelling error:
<img width="432" alt="screen shot 2017-12-27 at 10 03 28" src="https://user-images.githubusercontent.com/910753/34384928-3b08867e-eaed-11e7-9276-94863072a86b.png">

Adding me as a step works for me. 👍

**Comment by Forbrig** [comment]
@RichardLitt Can you invite me and add a _pt-br_ Readme file?

**Comment by RichardLitt** [comment]
Done. 

In the future, hit me up on gitter.im or by email, to keep notifications down. Let's add this to the CoC.

**Comment by eamanu** [comment]
@RichardLitt when I try to translate, gitlocalize say me: Content `You need to upgrade the plan to translate`. and don't save the translation

**Comment by Forbrig** [comment]
> @RichardLitt when I try to translate, gitlocalize say me: Content You need to upgrade the plan to translate. and don't save the translation

Same here, @sotayamashita can you help us?

**Comment by sotayamashita** [comment]
@RichardLitt Is it possible?

I can give you our free OSS plan if you can follow these conditions:

- The OSS projects should be non-profitable
- Up to 5 repositories
- Put a note that GitLocalize is used for a localization and its project link on README.md
- If you have a website for a project, put a GitLocalize logo onto the site

**Comment by jpmakangiras** [comment]
@RichardLitt can you invite me and add id Readme file on Gitlocalize?

**Comment by RichardLitt** [comment]
@sotayamashita We do that - it's only this repo, which is only OSS, no website yet, and which has no monetary incentive. Happy to add a link.

**Comment by sotayamashita** [comment]
@RichardLitt Thanks. I will create the PR.

**Comment by Forbrig** [comment]
@sotayamashita I can't edit others submissions, I always get "Content seems not same as original content". I'm the moderator of that pt-br translation, can you help me?

**Comment by IrvanFza** [comment]
Hi @k88hudson @jpmakangiras ... I would like to help too for Bahasa Indonesia translation. Can you add me to GitLocalize repo?

**Comment by sotayamashita** [comment]
Hi forks.

GitLocalize is unfortunately reaching its end of service soon.

**Comment by qWici** [comment]
@RichardLitt @k88hudson 


> Hi forks.
> 
> GitLocalize is unfortunately reaching its end of service soon.

Migrate to Crowdin?)

**Comment by RichardLitt** [comment]
Thanks, @sotayamashita, for all of your work.

CrowdIn makes sense for me.

**Comment by Pedro-Mendes** [comment]
Hey @Forbrig is the brazilian portuguese translation still in development? I'd like to participate too! How can I help?
@wviana @RichardLitt

**Comment by Forbrig** [comment]
Sure @Pedro-Mendes lets resume when it migrate to CrowdIn, you are welcomed to join!

**Comment by MuratGundes** [comment]
Hello guys. I'd like to help with the translation to Turkish.

**Comment by RichardLitt** [comment]
I've submitted a request to CrowdIn for a free project for this repo. I'll follow back up when I hear back.

**Comment by RichardLitt** [comment]
@k88hudson Looks like you need to enable it, not me. Can you sign up for CrowdIn and then enable the integration?

**Comment by RichardLitt** [comment]
@Forbrig No updates from me. I'm waiting on @k88hudson.

**Comment by opeerator** [comment]
Hey there @RichardLitt I think It wouls be cool to add Farsi as well

**Comment by RichardLitt** [comment]
Ping @k88hudson.

**Comment by qWici** [comment]
Whoa, this issue is still open. When I created it, I didn't think the discussion would last for 3 years. But it would be cool if we did get it over with.

**Comment by RichardLitt** [comment]
Same. I've tried pinging Kate elsewhere a few times. Not sure what else to do, short of forking this.

**Comment by HorseLord-Abrek** [comment]
Greetings, if translation process is still ongoing I can contribute to Turkish translations. As far as I can see there are no mods for Turkish so I can take the role.

**Comment by RichardLitt** [comment]
Works for me!

**Comment by HorseLord-Abrek** [comment]
One thing is, "Add Language" button won't show at GitLocalize. Should I fork the project and create a pull after I conduct translation myself or can you add the Turkish option to this repo's GitLocalize page?

**Comment by RichardLitt** [comment]
I've added you as the moderator for that language. :) Try now?

**Comment by HorseLord-Abrek** [comment]
Yup, right now I can see myself registered as moderator but file seems to be corrupted, it says translation is 100% and when I try to reach it, it gives 404 error.

**Comment by RichardLitt** [comment]
Not sure what to do, there. @sotayamashita Am I right that Gitlocalize is no longer being updated? Is there a way to pay you for your help?

**Comment by HorseLord-Abrek** [comment]
I will be waiting to see if there's a solution and if not I can find an alternative way. Both ways I am pretty eager to translate it into Turkish so it's no problem :) Thanks for your time.

**Comment by qWici** [comment]
@RichardLitt 
Why did we give up Crowdin? You can apply for a [free license for open source](https://crowdin.com/page/open-source-project-setup-request)

**Comment by RichardLitt** [comment]
See https://github.com/k88hudson/git-flight-rules/issues/162#issuecomment-432202030.

**Comment by HorseLord-Abrek** [comment]
It seems there is no response so I will start working on the file offline with my own translation tool, is it okay to proceed that way @RichardLitt ?

**Comment by RichardLitt** [comment]
A day may be a bit short. I would wait a week or two. You’re free to start
translating now, though!

**Comment by VietThan** [comment]
Just putting here the Crowdin requirements:

> 1. You have created a translation project in Crowdin.
2. The project is licensed under an approved license from an open-source initiative.
3. Your project's source code is publicly available for download.
4. You do not have any commercial products related to the open-source project you are requesting a license for.
5. You are the project lead.
6. You have been working on your open-source project for at least three months.
7. You have an active community of collaborators.
8. You keep the "News" section of your website up to date.
9. You release updated builds on a regular basis.

I don't think we fit a few of these requirements

**Comment by RichardLitt** [comment]
Yeah. I also can't do anything at the moment as far as submitting it to Crowdin.

**Comment by HorseLord-Abrek** [comment]
In fact I uploaded it to Crowdin as a private project and started working on it, because of the excessive workload I could not finish it yet but still got some progress.

**Comment by qWici** [comment]
@RichardLitt maybe you can make fork of this project and add to Crowdin. After that we can just merge updates to origin repo.

**Comment by RichardLitt** [comment]
I don't want to take on that extra responsibility. Happy to merge updated translations, though.

**Comment by Luminous-Coder** [comment]
Just curious. The readme asks people to use GitLocalize to translate, but forking and PR are still using now?
What to use to help with translation?

**Comment by RichardLitt** [comment]
I'm unsure. I don't have the resources to reevaluate GitLocalize, but let me know if we're not using it and we can remove that recommendation.
